File: Conexao.java
import java.sql.Connection;
import java.sql.DriverManager;

public class Conexao {

    public static Connection conectar() {
        try {
            String url = "jdbc:mysql://localhost:3306/zlar";
            String user = "root";
            String password = "";

            Connection conn = DriverManager.getConnection(url, user, password);
            return conn;

        } catch (Exception e) {
            System.out.println("Erro na conexão: " + e.getMessage());
            return null;
        }
    }
}

File: conexao.php
<?php

$host = "localhost";
$user = "root";
$password = "";
$db = "zlar";

$conexao = new mysqli($host, $user, $password, $db);

if ($conexao->connect_error) {
    die("Erro na conexão: " . $conexao->connect_error);
}
?>